### DISCUSSION TITLE
部署流程优化以及登录安全性问题

### DISCUSSION THREAD

**Discussion by atogumo** [gethexon/hexon#26] [General]
1. 部署流程优化
现在执行pnpm run setup之后需要在控制台手动输入端口号、博客路径等信息，我想尝试在Vercel部署该程序，但是Vercel不能输入命令，所以我想能否增加一个配置文件或一个执行默认配置的命令，以便一键部署。

2. 登录安全性
我看到login时传递的是明文密码，这是否会有安全隐患，比如当前网络环境是公共场所或使用了第三方代理，有没有必要加入加密算法。

最后提一点就是有些说明感觉有点简略、有歧义，比如自定义脚本，昨晚试了好几次才意识到说明里的路径指的是把脚本文件放博客并填写文件相对于博客目录的相对路径，不介意的话我是否能够更改README以及程序内的说明内容，当然，使用的是pull request的方式。

**Comment by YuJianghao** [comment]
1. 是个好主意
2. https 是安全的，http 时会启用加密，但也防君子不防小人
3. 文档确实有些简单，如果可以帮忙修改，那再好不过了 ❤️

**Reply by atogumo** [reply]
关于第一点，我看issue提供的思路是env file，今晚我尝试过了，可以实现，env的内容如果作为初始内容则必须要提示用户初次登录修改密码，这增加了工作量。
还有我需要道个歉因为试了一晚上我才发现Vercel并不支持部署服务端🤣，难怪部署上去总是找不到后端服务：
> The Vercel platform is serverless and designed for static frontends and [Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions). **There is no running server.**

所以是否需要“一键部署”可能还需要再斟酌一下

**Reply by iamliuzy** [reply]
另外，在使用 GitHub Pages 部署的 Hexo 博客中能否使用 Hexon ？

**Comment by atogumo** [comment]
> 另外，在使用 GitHub Pages 部署的 Hexo 博客中能否使用 Hexon ？

可以使用，Hexon是一个在线的markdown编辑器，附带一些简单的管理功能，部署不受Hexon影响

**Reply by iamliuzy** [reply]
好的谢谢
